components/Controllers/AuctionPageContent/AuctionGraph.tsx

```diff
@@ -171,23 +171,24 @@ export function AuctionGraph({
   }, [oracleInfo, timestampAndPricesAllTime, auction.auctionAssetContract.id]);
 
   const startLabel = useMemo(() => {
-    const floorPrice =
+    const floorPriceString =
       oracleInfo[auction.auctionAssetContract.id].price.toFixed(4);
+    const floorUSDPriceString = floorUSDPrice.toFixed(2);
```

Comment:
Is there a reason we're not using this? https://github.com/with-backed/papr-interface/blob/main/lib/numberFormat.ts#L35

Reply:
used it!